### ISSUE TITLE
why this mod can't use with croptopia ?

### ISSUE THREAD

**Issue by Kirinat0** [coolsimulations/SurvivalPlus-Lightsabers#3]
i install this mod tomy modpack and i get some bad think my croptopia mod can't drop the seed if i use this mod and idon't know how to do i like your mod so much is actually cool mod bro. u can fix is right?

**Comment by coolsimulations** [comment]
Before I can comment on this, is this Forge or Fabric? And what Minecraft version is this issue occurring on?

**Comment by Kirinat0** [comment]
i use minecraft 1.16.5 fabric

**Comment by coolsimulations** [comment]
As far as I can tell this seems to be a problem with Croptopia. Tried finding/harvesting seeds in Plains, Swamps and Taiga without success with Croptopia 1.4.0 for 1.16. Farmer Villager Trades didn't seem to work either. I tried with and without SurvivalPlus and SurvivalPlus Lightsabers and neither worked, however, 1.17 works. If you provide more specific information and/or test your issue without SurvivalPlus or SurvivalPlus Lightsabers and find seed drops works, I will re-open this issue.
